catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/JpegService.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.imaging.chip.transformer;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+
+/**
+ * Service to convert a BufferedImage into JPEG image data.
+ */
+public interface JpegService {
+
+    byte[] createJpeg(BufferedImage bufferedImage) throws IOException;
```

Comment:
It may be obvious what this is doing, but some javadoc on this interface would be ideal